kubesignin/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: A Kubesignin chart for Kubernetes
 name: kubesignin
-version: 0.4.0
-keywords: [skyscrapers, kubesignin, dex, auth, oidc]
+version: 0.5.0
```

Comment:
I would rather bump it to 1.0.0 already, as this is a major change

Reply:
Ok, will do.

However we should rework the whole chart in a next release, this one doesn't really adhere to best practices.